axon-framework/monitoring/tracing.md

```diff
@@ -0,0 +1,535 @@
+{% tab title="Spring Boot Bean" %}
+
+```java
+@Configuration
+public class AxonConfiguration {
+
+    @Bean
+    public SpanFactory spanFactory() {
+        return new MultiSpanFactory(
+                Arrays.asList(
+                        LoggingSpanFactory.INSTANCE,
+                        OpenTelemetrySpanFactory
+                                .builder()
+                                .build()
+                )
+        );
+    }
+}
+```
+
+{% endtab %}
+{% tab title="Axon Configurer" %}
+
+```java
+public class AxonConfigurer {
+
+    public void configure(Configurer configurer) {
+        configurer.configureSpanFactory(configuration -> new MultiSpanFactory(
+                Arrays.asList(
+                        LoggingSpanFactory.INSTANCE,
+                        OpenTelemetrySpanFactory.builder().build()
+                )
+        ));
+    }
+}
+```
+
+{% endtab %}
+
+Whenever a span is now requested by the framework to be created, the factory calls a span that contains all spans of the
+configured factories and acts like a single one.
+
+## Features
+
+The configured `SpanFactory` is responsible for creating spans when the framework requests it.
+The framework specifies the type of span,
+the name, and a message that triggered the span (if any, it's not required). The framework can request
+the span types defined in the following table:
+
+| Span Type     | Description                                                                                      | 
+|---------------|--------------------------------------------------------------------------------------------------|
+| Root trace    | Create a new trace entirely, having no parent.                                                   |
+| Dispatch span | A span which is dispatching a message.                                                           |
+| Handler span  | A span which is handling a message. Will set the span that dispatched the message as the parent. |
+| Internal span | A span which specified something internal. It's not an entry or exit point.                      |
+
+A trace generally consists of multiple spans with different types, depending on the functionality.
+When a message is dispatched and handled by another process or thread, the handling trace is related to the dispatching
+trace.
+This handling trace becomes part of the root trace unless it's an asynchronous invocation. In that case, it will be
+linked to it instead,
+so tooling can refer to the related traces but the traces don't become so large they are unreadable.
+
+The following functionality in Axon Framework is traced in addition to the tracing capabilities already provided by the
+standard of your choice:
+
+- Dispatching and handling of commands
+- Publishing of events
+- Handling of events by event processors
+- Dispatching and handling of queries
+- Scheduling and invocation of deadlines
+- Creation of snapshots
+- Loading of the aggregate, including time spent waiting for a lock
```

Comment:
How about we link to the chapters of each in this list?
So, we link to Command dispatching/handling, Event publication, Event Processors, Query dispatching/handling, deadlines, snapshots, [repositories](https://docs.axoniq.io/reference-guide/axon-framework/axon-framework-commands/configuration#command-model-repositories), and sagas.

Reply:
Agreed, adding this